### PR TITLE
Remove "Invite to share screen" feature

### PR DESCRIPTION
**Pull request by finn0**
closes https://github.com/mozilla/sumo-project/issues/437.

### CODE REVIEW

**Review by LeoMcA — COMMENTED** [review]
Looks good! I think we can also remove the `screen_share` permission from the User model: https://github.com/mozilla/kitsune/blob/190e1cce0a8c69559c66288dd6db6484a1f2d2dd/kitsune/users/models.py#L118-L119

You'll need to generate a migration with `./manage.py makemigrations` after that.

I'd also squash the two commits into one, it doesn't make sense to have the tests without the feature.

**Comment by finn0** [comment]
Why am I getting `Not Found: /en-US/kb/common-forum-responses` error, when trying to use `Common responses` feature? Looks like page is not available because, when I opened `http://0.0.0.0:8000/en-US/kb/common-forum-responses` URL, I get `Page Not Found (404)` error.

**Comment by finn0** [comment]
Err, looks like I tagged a wrong issue ticket in git's commit message, force pushing once again.

**Comment by LeoMcA** [comment]
> Why am I getting Not Found: /en-US/kb/common-forum-responses error, when trying to use Common responses feature? Looks like page is not available

Indeed, you can create the kb article locally by going to `/kb/new` and filling in `common-forum-responses` as the slug in the appropriate field

**Comment by akatsoulas** [comment]
@LeoMcA @finn0 this needs a rebase probably b/c it has conflicts. Apart from that, is it good go?

**Comment by akatsoulas** [comment]
Even if it's ready, pls do not merge before Wed 23rd

**Comment by finn0** [comment]
> this needs a rebase probably b/c it has conflicts.

I'll resolve it and force push over this branch.

> Apart from that, is it good go?

After resolving the conflict, I'll test it locally, once again.

**Comment by finn0** [comment]
@LeoMcA can you take a look into PR, once again? I've resolved the merge conflict and tested this locally.

earlier now

Now, there is no 'Invite to share screen' option.

File: kitsune/users/models.py [hunk 1]
```diff
@@ -115,7 +115,6 @@ class Meta(object):
         permissions = (
             ("view_karma_points", "Can view karma points"),
             ("deactivate_users", "Can deactivate users"),
-            ("screen_share", "Can screen share"),
```
Comment (LeoMcA) [inline kitsune/users/models.py:118]: You need to run `./manage.py makemigrations` given this change, and commit the resulting file.

**Review by LeoMcA — COMMENTED** [review]
Looks good! Just one small thing:

File: kitsune/users/models.py (hunk 1 above)
Comment (finn0) [reply kitsune/users/models.py:118]: I've committed migration file. Can you take a look into pull request, once again?

**Comment by LeoMcA** [comment]
Fantastic, thank you!

**Comment by akatsoulas** [comment]
Nice work everyone!

**Pull request merged**